msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Build apps for Teams meeting stage
+author: v-sdhakshina
+description: In this article, learn how to build meeting apps API references that are available for Teams client.
+ms.topic: conceptual
+ms.author: v-sdhakshina
+ms.localizationpriority: medium
+ms.date: 04/07/2022
+---
+
+# Build apps for Teams meeting stage
+
+Share to stage allows users to share an app to the meeting stage from the app side panel in an ongoing meeting. This sharing is interactive & collaborative as opposed to passive screen sharing.
```

Comment:
This page should touch on "why" it is collaborative. If an app doesn't have web sockets, Azure Fluid Relay, Live Share, or some other RTC protocol for example there is a good chance the app is not equipped for collaboration on meeting stage. 

Also, a more clear reference to Live Share would be helpful. There is also a section in the Live Share overview page that gives a more clear overview of sockets vs. Azure Fluid Relay vs. Live Share that may be relevant in this page.

Reply:
created WI https://dev.azure.com/domoreexp/MSTeams/_workitems/edit/2694776